src/LeanplumAnalyticsEventForwarder.js

```diff
@@ -233,6 +233,13 @@
         }
 
         function setLeanPlumEnvironment() {
+            var TWO_HOURS = 2*60*60;
+            Leanplum.useSessionLength(forwarderSettings.sessionLength || TWO_HOURS);
+
+            if (forwarderSettings.enableRichInAppMessages) {
```

Comment:
Not that you would know this, but strangely, our server will return "True" or "False" rather than a boolean `true` or `false`. Almost forgot myself!  Here's an example from [another integration](https://github.com/mparticle-integrations/mparticle-javascript-integration-appboy/blob/master/src/AppboyKit-dev.js#L308-L309).

```suggestion
            if (forwarderSettings.enableRichInAppMessages === 'True') {
```